### PR TITLE
Add Debian 11 (bullseye) as platform

### PR DESCRIPTION
**Pull request by flyinggecko**
Hi,

This PR adds Debian 11 as target platform for this role.

Bests

### CODE REVIEW

File: meta/main.yml [hunk 1]
```diff
@@ -26,6 +26,9 @@ galaxy_info:
       versions:
         - bionic
         - focal
+    - name: Debian
+      versions:
+        - 11
```
Comment (tobiashuste) [inline meta/main.yml:31]: `ansible-lint` expects a string here.

```suggestion
        - "11"
```

**Review by tobiashuste — CHANGES_REQUESTED** [review]
Dear @flyinggecko, thanks for providing this. Very much appreciated!

Would you please add debian 11 to the GitHub Actions pipelline as well. This can be done by adding `debian11` to this list:

https://github.com/hifis-net/ansible-role-gitlab/blob/657524215d6a8b0b0cdfcb14d2d7980dc0ea8e3f/.github/workflows/ci.yml#L56-L60

Thanks :smiley:

**Comment by flyinggecko** [comment]
Hi,

I've added all the requested changes.

Bests ;)

**Review by tobiashuste — APPROVED** [review]
Thanks, @flyinggecko :rocket: Will create a new release soon.

**Pull request merged**